test/profiler/test_profiler.py

```diff
@@ -1050,6 +1051,7 @@ def trace_handler(p):
             self.assertEqual(test_schedule(step), test_schedule_expected_outputs[step])
 
     def test_kineto_profiler_multiple_steppers(self):
+        _profile_using_dynolog(True)
```

Comment:
What happens if the test is repeated? Perhaps there should be a context manager for testing?

Reply:
Could you share a scenario where the test would run twice within the same process? Perhaps, your concern is about `_profile_using_dynolog` being idempotent?

Reply:
All the test run one after the other in a single process. So setting this flag without ever removing it will change the behavior of all the tests that run after this one.
This is not ok as this can easily lead to non-reproducible errors in CI depending on the order the tests run in!